### PR TITLE
Testing example

### PR DESCRIPTION
**Pull request by gvdongen**

### CODE REVIEW

File: templates/typescript-testing/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+COPY --chown=node:node . .
+
+RUN npm run build
+
+RUN npm prune --production
+ENV NODE_ENV production
+
+EXPOSE 9080
+USER node
+CMD ["dumb-init", "node", "./dist/app.js"]
```
Comment (slinkydeveloper) [inline templates/typescript-testing/Dockerfile:21]: no need for this

File: templates/typescript-testing/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Hello world - TypeScript example
+
+Sample project configuration of a Restate service using the TypeScript SDK.
+
+Have a look at the [TypeScript Quickstart guide](https://docs.restate.dev/get_started/quickstart?sdk=ts) for more information on how to use this project.
```
Comment (slinkydeveloper) [inline templates/typescript-testing/README.md:5]: maybe change this text

File: templates/typescript-testing/test/restate_test_environment.ts [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+    return restateHttpServer;
+}
+
+// Prepare the restate testcontainer
+async function prepareRestateTestContainer(
+    restateServerPort: number
+): Promise<StartedTestContainer> {
+    const restateContainer = new GenericContainer(
+        "docker.io/restatedev/restate:1.0.1"
```
Comment (slinkydeveloper) [inline templates/typescript-testing/test/restate_test_environment.ts:40]: just use `latest` here.

File: templates/typescript-testing/tsconfig.json [hunk 4]
```diff
@@ -0,0 +1,110 @@
+{
```
Comment (slinkydeveloper) [inline templates/typescript-testing/tsconfig.json:1]: is this aligned with the current hello world example?

**Review by slinkydeveloper — COMMENTED** [review]
looks good to me.

Perhaps add it to the test script, and to the auto bump sdk scripts, but i don't think we need to add it to the zips.

File: templates/typescript-testing/tsconfig.json (hunk 4 above)
Comment (gvdongen) [reply templates/typescript-testing/tsconfig.json:1]: Yes, the file is aligned

**Pull request merged**